eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOstdinTest.java

```diff
@@ -128,6 +128,34 @@ public void stdinEmptyTest() {
         );
     }
 
+    @Test
+    public void stdinfewOneLineTest() {
+        String expected1 = "first-line";
+        String expected2 = "second-line";
+        String expected3 = "third-line";
```

Comment:
@includealex It would be better to use variables name without numbers. I think we can call these variables `first`, `second` and `third`.